src/bridgetree.rs

```diff
@@ -411,8 +396,20 @@ impl<A> AuthFragment<A> {
         }
     }
 
+    pub fn position(&self) -> Position {
+        self.position
+    }
+
+    pub fn altitudes_observed(&self) -> usize {
+        self.altitudes_observed
+    }
+
+    pub fn values(&self) -> &Vec<A> {
```

Comment:
You almost never want to return `&Vec<_>`.
```suggestion
    pub fn values(&self) -> &[A] {
```